src/status_im2/contexts/chat/composer/handlers.cljs

```diff
@@ -18,10 +18,12 @@
    {:keys [text-value focused? lock-selection? saved-cursor-position gradient-z-index]}
    {:keys [height saved-height last-height opacity background-y gradient-opacity container-opacity]
     :as   animations}
-   {:keys [max-height] :as dimensions}]
+   {:keys [max-height] :as dimensions}
+   show-floating-scroll-down-button?]
   (reset! focused? true)
   (rf/dispatch [:chat.ui/set-input-focused true])
   (reanimated/animate height (reanimated/get-shared-value last-height))
+  (reset! show-floating-scroll-down-button? false)
```

Comment:
Please move this line to the bottom of the function just for sake of organization

Reply:
Will do before merging